{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/403_csrf.html

```diff
@@ -0,0 +1,14 @@
+{% raw %}{% extends "base.html" %}
+
+{% block title %}Forbidden (403){% endblock title %}
+{% block content %}
+  <h1>Forbidden (403)</h1>
+  <p>
+    {% if exception %}
+      {{ exception }}
+    {% else %}
+      You're not allowed to access this page.
```

Comment:
Should we change the content? Because it's a bit difficult to see that it was a CSRF failure here and it was not a regular 403.

```suggestion
      CSRF verification failed. Request aborted.
```

Reply:
In the original issue, it says that the message "CSRF verification  failed..." was supposed to be "enexpected". So I was not sure if I should add this error message to the new file or keep it the way it is. But yeah if you want me to change it, I'll do it.

Reply:
I see. Yes I guess one could argue either way... I'm not sure showing what I suggest is any more helpful for users actually. Let's leave it like this for now.